src/test/java/de/retest/recheck/review/ignore/matcher/ElementClassMatcherTest.java

```diff
@@ -51,16 +51,17 @@ void should_not_match_any_class_value_if_class_key_is_absent() throws Exception
 
 	@Test
 	void should_match_multiple_equal_class_values() throws Exception {
-		final Element oneTwoOnone = mockElementWithClassValue( "one two onone" );
+		final Element oneTwoOnone = mockElementWithClassValue( "one two" );
 
 		final ElementClassMatcher cut = new ElementClassMatcher( oneTwoOnone );
 
-		final Element one = mockElementWithClassValue( "one" );
+		final Element one = mockElementWithClassValue( "oneone" );
 		final Element two = mockElementWithClassValue( "two" );
 		final Element oneTwo = mockElementWithClassValue( "one two" );
-		final Element twoOne = mockElementWithClassValue( "two one" );
+		final Element twoOne = mockElementWithClassValue( "boo two one" );
 
-		assertThat( cut ).accepts( one, two, oneTwo, twoOne );
+		assertThat( cut ).accepts( oneTwo, twoOne );
+		assertThat( cut ).rejects( one, two );
```

Comment:
To recap, if the user specifies (as above):

```
matcher: class=one two
```

It should filter elements with classes `one two` and `boo two one`, but not `oneone` and `two`? That is, elements must at least contain all classes from the filter?

Reply:
That's IMHO what I would intuitively expect. If I want to use different classes, I would create different rules.

Reply:
We discussed this when implementing, but I do not remember our decision.. 🤔 

I took some time to think about this, but I now agree with @roesslerj.

Consider the following example:
```
<div class="one two">
```

We currently have this the other way round, an element must specify a subset of the filter, but cannot contain a superset. 

this will never match against:
```
matcher: class=one
```

The test does it the other way round. It assumes someone writes the filter:
```
matcher: class=one two onone
```
and checks against the following elements:
```
<div class="one">
<div class="two">
<div class="one two">
<div class="two one">
```

---

Coming from Selenium and/or css selectors, one would assume:

1. `By.className("one")` matches
2. `By.className("two")` matches
3. By.className("one two") Selenium cannot handle multiple classes... or I am doing something wrong...
4. `.one` matches
5. `.two` matches
6. `.one.two` matches
7. `.two.one` matches

Using this for our filters:
1. `class=one` matches
2. `class=two` matches
3. `class=one two` matches
4. `class=two one` matches
5. `class=one two onone` does not match

Reply:
So ... do you agree or is there still something to do?

Reply:
If there's nothing todo, please approve somebody...